src/Sulu/Component/SmartContent/ContentType.php

```diff
@@ -275,7 +275,7 @@ public function getContentData(PropertyInterface $property)
         $limit = (\array_key_exists('limitResult', $filters) && $configuration->hasLimit()) ?
             $filters['limitResult'] : null;
         $options = [
-            'webspaceKey' => $property->getStructure()->getWebspaceKey(),
+            'webspaceKey' => $this->requestAnalyzer->getWebspace()->getKey(),
```

Comment:
the article structure is not assigned to any webspace. therefore `$property->getStructure()->getWebspaceKey()` will always be null if we use a smart-content property inside of an article template